lesson_1.1/src/App.js

```diff
@@ -5,32 +5,34 @@ import AddTodoForm from "./AddTodoForm";
 
 function useSemiPersistentState() {
   const [todoList, setTodoList] = useState(
-    localStorage.getItem ('savedTodoList')? JSON.parse(localStorage.getItem('savedTodoList')) : [] 
-  );
-
-useEffect(() => {
```

Comment:
this maybe a personal preference thing so I'd ask your advisor, but I prefer and usually see useEffect split up into multiple lines like you originally had it because it makes the code a bit more readable than everything written out on one line. Less lines of code isn't always better code